cli_tools/gce_windows_upgrade/upgrade_script.ps1

```diff
@@ -56,14 +85,14 @@ online disk noerr
 
   # Run upgrade script from the install media.
   Set-ExecutionPolicy Unrestricted
-  Set-Location "$($script:install_media_drive)/Windows_Svr_Std_and_DataCtr_2012_R2_64Bit_English"
+  Set-Location "$($script:install_media_drive)/$script:install_folder"
   ./upgrade.ps1
-  $new_ver=[System.Environment]::OSVersion.Version
-  if ("$($ver.Major).$($ver.Minor)" -eq "6.3")
+  $new_ver=(Get-ItemProperty "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion").ProductName
+  if ($new_ver -eq $script:expected_new_version)
   {
     Write-Host "GCEMetadataScripts: post-upgrade step is done."
```

Comment:
Why does this use a prefix of `GCEMetadataScripts`? Shouldn't it have a prefix of the upgrade process?

Reply:
The whole script is a startup script. This script will output some important status, like "GCEMetadataScripts: Beginning upgrade startup script." Our workflow needs to watch this message to ensure the machine is boot and started executing the upgrade script. We can name it with any prefix, but this prefix indicates everything happens due to the launch of GCEMetadata startup script in my opinion.

Reply:
If upgrade_script.ps1 is launched using metadata startup scripts then any output from the script is captured by the GCEMetadataScripts binary and it includes the logging prefix of "GCEMetadataScripts: ".

So my concern is that the output in the serial port will look like:
"GCEMetadataScripts: GCEMetadataScripts:  post-upgrade step is done."

Reply:
You are completely right, it's duplicated. Removed it from this script.